doc/lsst.ip.diffim/tasks/lsst.ip.diffim.DetectAndMeasureScoreTask.rst

```diff
@@ -0,0 +1,60 @@
+.. lsst-task-topic:: lsst.ip.diffim.DetectAndMeasureScoreTask
+
+#########################
+DetectAndMeasureScoreTask
+#########################
+
+DetectAndMeasureScoreTask performs source detection and execution of measurement plugins in difference images.
+
+.. _lsst.ip.diffim.DetectAndMeasureScoreTask-description:
+
+Description
+===========
+
+This task detects sources in score-like difference images, such as the ones obtained with pre-convolution
+image differencing, or ZOGY "S" images.
+This means that cross-convolution is not needed, and the task obtains maximum-likelihood source candidates
+in a single step. 
+Once candidate sources are obtained this task performs the same operations as DetectAndMeasureTask, 
+for further reference consult the documentation of DetectAndMeasureTask.
```

Comment:
Please link to said tasks when mentioned in text. Do this for the other files above, too, please.

```suggestion
Once candidate sources are obtained this task performs the same operations as `~lsst.ip.diffim.DetectAndMeasureTask`;
for further reference consult the documentation of `~lsst.ip.diffim.DetectAndMeasureTask`.
```